example_config.yaml

```diff
@@ -37,4 +37,15 @@ subscriptions:
 # sync data model
 data_modeling:
     - space: cc_plant
-      lakehouse_abfss_prefix: abfss://FabricCogniteDemo@onelake.dfs.fabric.microsoft.com/CogniteLakehouse.Lakehouse
+      lakehouse_abfss_prefix: ${LAKEHOUSE_ABFSS_PREFIX}
+
+source:
+    abfss_prefix: ${LAKEHOUSE_ABFSS_PREFIX}
```

Comment:
In you verification step in the PR, they reference settings that do not exist, such as abfss_event_table_path. Assuming they changed after PR feedback, but you should probably also update the verification steps.